x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/components/empty_state.tsx

```diff
@@ -11,7 +11,7 @@ import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
 
 import { i18n } from '@kbn/i18n';
 
-import { API_DOCS_URL } from '../../../routes';
+import { docLinks } from '../../../../shared/doc_links';
```

Comment:
I am not very happy with this import and would prefer to use a property for the component. But considering the number of components to be updated, it would be too long and risky to do it like this.

 it will be good enough to fix the issue before 8.16.0 is released and we can revisit it later.